src/collective/cover/static/layout_edit.js

```diff
@@ -129,6 +132,11 @@
              * See: http://stackoverflow.com/a/2117523/2116850
              **/
             generate_uuid: function () {
+              // Improve randomness. See: http://stackoverflow.com/a/8809472/2116850
+              self.timestamp = new Date().getTime();
+              if (window.performance && typeof window.performance.now === 'function') {
+                self.timestamp += performance.now();; // use high-precision timer if available
+              }
```

Comment:
I'm not sure if the double `;` are needed.. just keep the same implementation of the stack overflow.